resource/css/skosmos.css

```diff
@@ -698,12 +793,14 @@ body {
   #alphabetical-menu > .list-group > .list-group-item {
     border: none !important;
   }
+  
+  /*** Sidebar ***/
+  /*
```

Comment:
Seems that quite a lot has been commented out here? Wouldn't it be better to remove these if they are not used?